commitlint.config.js

```diff
@@ -1,8 +1,24 @@
-// @noflow
+const fs = require("fs");
+
+function fetchComponentDirs() {
+  const dirs = fs
+    .readdirSync("./packages/orbit-components/src/", { withFileTypes: true })
+    .filter(dirent => dirent.isDirectory())
```

Comment:
currently `__typetests__` , data,  folders are also valid scopes, I think it shouldn't be